pkg/hoist/hoist_launchable.go

```diff
@@ -374,10 +379,22 @@ func (hl *Launchable) Executables(
 				ExtraEnv:         map[string]string{launch.EntryPointEnvVar: relativePath},
 				NoLimits:         hl.ExecNoLimit,
 				CgroupConfigName: hl.CgroupConfigName,
-				PodID:            &hl.PodID,
 				CgroupName:       hl.CgroupName,
 				RequireFile:      hl.RequireFile,
 			}
+			if *IncludePodIDArg {
+				p2ExecArgs = p2exec.P2ExecArgs{
```

Comment:
I believe you can dry this out a bit with
```
if *IncludePodIDArg {
    p2ExecArgs.PodID = &hl.PodID
}```
and the rest of the fields will carry over from the preceding initialization.